cli.ts

```diff
@@ -1752,10 +1770,24 @@ yargs(hideBin(process.argv)) // eslint-disable-line no-unused-expressions
     command: 'pull [source]',
     desc: 'Pull analytics wrappers from Avo workspace',
     builder: (yargs) =>
-      yargs.option('branch', {
-        describe: 'Name of Avo branch to pull from',
-        type: 'string',
-      }),
+      yargs
+        .option('branch', {
+          describe: 'Name of Avo branch to pull from',
+          default: 'main',
+          type: 'string',
+        })
+        .option('f', {
+          alias: 'force',
+          describe:
+            'Proceed ignoring the unsupported features for given source',
+          default: false,
+          type: 'boolean',
+        })
+        .option('forceFeatures', {
+          describe: 'Optional comma separated list of features to force enable',
+          default: undefined,
+          type: 'string',
```

Comment:
How do we know the features available? Does the cli print the availalbe feature list somehow?